app/views/reading_groups/show.html.erb

```diff
@@ -3,22 +3,22 @@
 </div>
 
 <p>
-  <strong>Description:</strong>
+  <strong><%= ReadingGroup.human_attribute_name('description') %></strong>
   <%= @reading_group.description %>
 </p>
 
 <hr />
 
-<h2>Readings</h2>
+<h2><%= Reading.model_name.human(count: 2) %></h2>
 
-<% if reading_groups_owner? @reading_group.id %><%= link_to 'New Reading', new_reading_group_reading_path(@reading_group,), class: 'btn btn-primary' %><% end %>
+<% if reading_groups_owner? @reading_group.id %><%= link_to "#{t('new.female')} #{Reading.model_name.human}", new_reading_group_reading_path(@reading_group,), class: 'btn btn-primary' %><% end %>
```

Comment:
We are going to discuss this soon. I do agree with you but fixing this will be a history of its own.